app/src/main/java/com/cortado/minionify/preference/PreferencesSingleton.java

```diff
@@ -0,0 +1,46 @@
+package com.cortado.minionify.preference;
+
+import android.content.SharedPreferences;
+
+import com.cortado.minionify.mode.AppMode;
+import com.cortado.minionify.mode.exceptions.ModeNotExistsException;
+import com.cortado.minionify.preference.exceptions.PreferenceOperationFailure;
+
+public class PreferencesSingleton {
+    private static PreferencesSingleton INSTANCE = null;
```

Comment:
not according to any naming convention, this is not a const (final) so shouldnt be all CAPS

Reply:
Irrelevent (no singleton